security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme_utils.inc

```diff
@@ -53,6 +53,27 @@ if(!function_exists('is_arrayset')){
 	}
 }
 
+if (!function_exists('getarraybyref')) {
```

Comment:
I know it can make things easier for portability but I'd really rather not have the same function redefined in several packages like this again. Too many bad memories of pfSense 2.2 and before with copy/pasted code.

Let's get this function into base and then only use this version of the package on 2.4.4 and later.

Reply:
so how do we get this version of the package into 2.4.3 ? there is no reason to restrict the new abilities to 2.4.4 just because of one little 'helper function'..? perhaps we can mark that part of the code 'to be removed after 2.4.4 gets released' ? it would imho be nice if this could be available for everyone in a few days time.

Reply:
well okay, removed from here.